src/common/lib/Messenger.js

```diff
@@ -33,7 +33,11 @@ export default class Messenger extends EventEmitter {
 
     this.removeAllListeners();
 
-    this.connection = new WebSocket(this.url);
+    try {
+      this.connection = new WebSocket(this.url);
+    } catch (err) {
```

Comment:
Yep, intentional here. If the websocket service is not properly configured we still want the UI to work right?